thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/AppView.js

```diff
@@ -4,10 +4,8 @@ function AppView(appModel) {
   this.currentActiveTab = undefined;
 }
 AppView.prototype = {
-
-  init : function() {
-    var self = this;
-    var tabSelectionEventHandler = function(e) {
+  init() {
+    var tabSelectionEventHandler = (e) => {
```

Comment:
this is es6's new arrow function expression. They do not bind their own `this` context and allow to avoid having to use `self`. ES6 syntax is now strongly preferred. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions